manifest.versions.json

```diff
@@ -1,5 +1,21 @@
 {
     "variables": {
+      "kb|ltsc2016|3.5-4.7.2":"kb4598243",
+      "lcu|ltsc2016|3.5-4.7.2":"http://download.windowsupdate.com/c/msdownload/update/software/secu/2021/01/windows10.0-kb4598243-x64_a96fed949c557064b0e105745a5524717ad72ab2.msu",
+      "kb|ltsc2019|3.5-4.7.2":"kb4586875",
+      "lcu|ltsc2019|3.5-4.7.2":"http://download.windowsupdate.com/c/msdownload/update/software/updt/2020/11/windows10.0-kb4586875-x64_a14cffeb8033542a5379d4c5a9c0c6fa17c3f939.msu",
+
+      "kb|ltsc2016":"kb4597247",
```

Comment:
Having two artifacts (kb and url) is not ideal IMO.  Parsing out the kb within the template is technically possible but I thought this was the more approachable solution.